core/services/editor/EditorBlockRegistrationManager.php

```diff
@@ -84,12 +85,13 @@ protected function populateEditorBlockCollection()
                 // FQCNs for classes to add (all classes within each namespace will be loaded)
                 apply_filters(
                     'FHEE__EventEspresso_core_services_editor_EditorBlockManager__populateEditorBlockCollection__collection_FQCNs',
-                    array(
-                        'EventEspresso\core\domain\entities\editor\blocks\common',
-                        'EventEspresso\core\domain\entities\editor\blocks\editor',
-                        'EventEspresso\core\domain\entities\editor\blocks\shortcodes',
-                        'EventEspresso\core\domain\entities\editor\blocks\widgets',
-                    )
+                    array()
+                    // array(
+                    //     'EventEspresso\core\domain\entities\editor\blocks\common',
+                    //     'EventEspresso\core\domain\entities\editor\blocks\editor',
+                    //     'EventEspresso\core\domain\entities\editor\blocks\shortcodes',
+                    //     'EventEspresso\core\domain\entities\editor\blocks\widgets',
+                    // )
```

Comment:
why this change?
this means we are no longer loading any blocks?

Reply:
those blocks don't exist in the branch this branch is based off of.  I think you forgot to move the config for those blocks into whatever branch you created for them.  I discovered this when I loaded this branch and there was an exception thrown.

Reply:
those aren't blocks, they're folders and I think phpStorm, or maybe Git, is deleting them because hey are empty...

might have to keep your changes for the time being until something gets added to each folder

Reply:
You can't commit empty folders in git. They aren't getting deleted. You may have them existing locally on your machine but they aren't being tracked.  Usually what I do when I want to create an "empty" folder structure to be tracked I'll add an _empty_`.gitignore` file to the folder.  

I realize these are paths, I was just carrying off your statement "this means we are not loading any blocks?", and I was simply pointing out the "blocks" I thought you were referring to don't exist on this branch and then I went on to say: _I think you forgot to move the config for those blocks into whatever branch you created for them_ because I was assuming at one point there _were_ blocks in these paths and you simply removed them and added them to a different branch (but didn't remove the config and restore the config - or in this case _path references_ - in that same branch).

So what do you want to do?  Keep these changes or add `.gitignore` so the folders are tracked?   imo I think we should only setup paths when there's actually something in them to process but I'll leave it up to you.